src/smtpclient.cpp

```diff
@@ -348,16 +352,20 @@ bool SmtpClient::login(const QString &user, const QString &password, AuthMethod
     return true;
 }
 
-bool SmtpClient::sendMail(MimeMessage& email)
+int SmtpClient::sendMail(MimeMessage& email)
```

Comment:
Changing the return type may potentially break backward compatibility. I would keep as it is now. There is already a `getResponse()` method which can be used to obtain the last `errorCode`.

Reply:
I am aware of it...
Sorry, I did not see the getResponseCode/Text...
It's a fine solution too...

Reply:
I am aware...
Sorry, I ddi not see getResponseCode/Text.
It' a great solution too...